src/explore-education-statistics-admin/src/components/wysiwyg.module.scss

```diff
@@ -56,8 +75,12 @@
     margin: 0;
     padding: 0;
     position: absolute;
-    right: 4px;
-    top: 2px;
+    right: 0;
+    top: 0;
     z-index: 1000;
   }
 }
+
+.preTextWrap pre {
```

Comment:
The style class name should relate to the component it is styling, not the containing styles